config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   namespace :api do
-    namespace :v1, defaults: { format: :json } do
```

Comment:
Why have you removed this line?

I think you can just move `mount Raddocs::App => "/docs"` to the top:

```ruby
Rails.application.routes.draw do
  mount Raddocs::App => "/api/v1/docs"
end
```